File: docs/user/en/get-started/example/query-demo-server.java
var entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.queryListOfSealedEntities(
			query(
				collection("brand"),
				filterBy(
					and(
						attributeStartsWith("name", "A"),
						entityLocaleEquals(Locale.ENGLISH)
					)
				),
				orderBy(
					attributeNatural("name", OrderDirection.ASC)
				),
				require(entityFetchAll())
			)
		);
	}
)

File: documentation/user/en/get-started/example/query-demo-server.cs
var entities = evita.QueryCatalog(
	"evita",
	session => {
		return session.QueryListOfSealedEntities(
			Query(
				Collection("Brand"),
				FilterBy(
					And(
						AttributeStartsWith("name", "A"),
						EntityLocaleEquals(new CultureInfo("en"))
					)
				),
				OrderBy(
					AttributeNatural("name", OrderDirection.Asc)
				),
				Require(EntityFetchAll())
			)
		);
	}
)